crypto/rand/drbg_lib.c

```diff
@@ -285,21 +360,34 @@ int RAND_DRBG_generate(RAND_DRBG *drbg, unsigned char *out, size_t outlen,
 }
 
 /*
- * Set the callbacks for entropy and nonce.  We currently don't use
- * the nonce; that's mainly for the KATs
+ * Set the RAND_DRBG callbacks for obtaining entropy and nonce.
+ *
+ * get_entropy() 
+ * 
+ *   This is a request to allocate and fill a buffer of size |min_len| <= size <= |max_len| 
+ *   (in bytes) which contains at least |entropy| bits of entropy. The buffers address is
+ *   to be returned in |*pout| and the number of collected randomness bytes as return value.
```

Comment:
Should that just say number of bytes (in pout)? It seems to give the impression that you want to know how much entropy it has as bytes.

Reply:
Can you also say that it returns 0 on error?